crates/katana/storage/db/src/tables.rs

```diff
@@ -223,8 +224,10 @@ tables! {
     /// storage change set
     StorageChangeSet: (ContractStorageKey) => BlockList,
     /// Account storage change set
-    StorageChangeHistory: (BlockNumber, ContractStorageKey) => ContractStorageEntry
+    StorageChangeHistory: (BlockNumber, ContractStorageKey) => ContractStorageEntry,
 
+    /// Stores the block number related to messaging service
+    MessagingInfo: (u64) => BlockNumber
```

Comment:
```suggestion
    MessagingCheckpoint: (MessagingCheckpointId) => BlockNumber
```

refer to comment below

Reply:
done